.github/workflows/ci-verification.yml

```diff
@@ -24,40 +24,25 @@ jobs:
     runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
     container:
       image: ghcr.io/microsoft/ccf/ci/default:build-25-07-2024
+    defaults:
+      run:
+        working-directory: tla
 
     steps:
       - uses: actions/checkout@v4
-      - run: |
+      - name: Install TLC dependencies
+        run: |
           sudo apt update
           sudo apt install -y default-jre
-          python3 ./tla/install_deps.py
-
-      - name: consistency/MCSingleNode.cfg
-        run: |
-          cd tla/
-          ./tlc.sh -workers auto consistency/MCSingleNode.tla -dumpTrace json MCSingleNode.json
-
-      - name: consistency/MCSingleNodeReads.cfg
-        run: |
-          cd tla/
-          ./tlc.sh -workers auto consistency/MCSingleNodeReads.tla -dumpTrace json MCSingleNodeReads.json
-
-      - name: consistency/MCMultiNode.cfg
-        run: |
-          cd tla/
-          ./tlc.sh -workers auto consistency/MCMultiNode.tla -dumpTrace json MCMultiNode.json
-
-      - name: consistency/MCMultiNodeReads.cfg
-        run: |
-          cd tla/
-          ./tlc.sh -workers auto consistency/MCMultiNodeReads.tla -dumpTrace json MCMultiNodeReads.json
+          python3 install_deps.py
 
-      - name: consistency/MCMultiNodeReadsAlt.cfg
-        run: |
-          cd tla/
-          ./tlc.sh -workers auto consistency/MCMultiNodeReadsAlt.tla -dumpTrace json MCMultiNodeReadsAlt.json
+      - run: ./tlc.py consistency/MCSingleNode.tla
```

Comment:
The combination of setting a default directory for the job, which is now supported in GHA, and terser CLI allows these to become one-liners again.